wagtail/admin/views/account.py

```diff
@@ -376,13 +376,6 @@ def get_context_data(self, **kwargs):
 
         context["show_password_reset"] = password_reset_enabled()
 
-        from django.contrib.auth import get_user_model
-
-        User = get_user_model()
-        context["username_field"] = User._meta.get_field(
-            User.USERNAME_FIELD
```

Comment:
Just curious as to why this removed.

Reply:
Probably because it's no longer used in the template, though I wonder if we should keep this in just in case people use it.

Reply:
Exactly, tell me if I should remove it or not

Reply:
Can we keep this please. It may be relied upon by others. Thanks @SebCorbin